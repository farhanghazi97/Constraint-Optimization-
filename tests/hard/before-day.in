meeting, m1
meeting, m2
meeting, m3
meeting, m4
domain, m1, before fri, hard
domain, m2, before thu, hard
domain, m3, before wed, hard
domain, m4, before tue, hard
