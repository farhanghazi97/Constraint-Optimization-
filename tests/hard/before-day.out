m1:tue 11am
m2:tue 11am
m3:tue 11am
m4:mon 9am
cost:0
